### PR TITLE
move keadm beta command to GA

### PR DESCRIPTION
**Pull request by gy95**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Part of #3879 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: keadm/cmd/keadm/app/cmd/beta/beta.go [hunk 1]
```diff
@@ -1,42 +0,0 @@
...
-	}
-
-	cmd.ResetFlags()
-
-	cmd.AddCommand(edge.NewJoinBetaCommand())
-	cmd.AddCommand(NewInitBeta())
-	cmd.AddCommand(NewManifestGenerateBeta())
-	cmd.AddCommand(newCmdConfig())
-	cmd.AddCommand(NewKubeEdgeResetBeta())
-
```
Comment (fisherxu) [inline keadm/cmd/keadm/app/cmd/beta/beta.go:40]: We can remain the beta sub-cmd here, in future we can add other beta cmd here.

**Review by fisherxu — COMMENTED** [review]
Overall looks good :)

File: keadm/cmd/keadm/app/cmd/beta/beta.go (hunk 1 above)
Comment (gy95) [reply keadm/cmd/keadm/app/cmd/beta/beta.go:40]: reverted, please review again

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**